OHIF/Viewers
# fix(Browser history): fixed NPE when navigating a study via browser history and history navigation is now available via the navigateHistory command

### Context
The community reported that navigating to a different study was not working for `window.history.pushState`. Furthermore it was confirmed that both `window.history.pushState` and the npm 'history' API do NOT trigger the react-router-dom - the react-router-dom is what OHIF uses to react to history changes.

As a result, it was decided to expose the browser history navigation via a command.



### Changes & Results

- fixed an NPE when navigating to a different study via the URL/browser history
- exposed browser history navigation via a command
- now a study can be navigated to via the `navigateHistory` command


### Testing

1. Add a button or hot key binding that will navigate to a different study. Something say like this (for a hot key binding)...
```
  {
    commandName: 'navigateHistory',
    commandOptions: {
      to:
        '/viewer?StudyInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125095722.1',
    },
    context: 'DEFAULT',
    label: 'nav study',
    keys: ['g'],
    isEditable: true,
  },
```
2. Launch OHIF to a study that is NOT specified in the button or key binding made in step 1.
3. Click the button or key that should change the browser history and study.


### Checklist

#### PR



  semantic-release format and guidelines.

#### Code

  etc.)

#### Public Documentation Updates



  additions or removals.

#### Tested Environment

- [x] Browser: Chrome 112.0.5615.138
  


[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch
